Assets/MixedRealityToolkit.SDK/Inspectors/UX/Collections/ScrollingObjectCollectionInspector.cs

```diff
@@ -0,0 +1,340 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please move to MixedRealityToolkit.SDK/Experimental/Inspectors/UX/Collections/ScrollingObjectCollectionInspector.cs